tutorials/others/ubuntu/tutorial.yml

```
@@ -0,0 +1,26 @@
+id: 78a3be56-5d51-4ec3-8629-0dd27c352ab5
+
+builder: Canonical
+
+tags:
+  - privacy
+  - open-source
+  - OS
```

```suggestion
  - DIY
```
(Tag "OS" is not in the Paolo's list)